laravel/app/Models/Configuration.php

```diff
@@ -35,7 +35,7 @@ public function loadConfigSettings()
                 $configValue = (int)$configValue;
             }
             if (!defined(strtoupper($config['configuration_key']))) {
-                define(strtoupper($config['configuration_key']), $configValue);
+                define(trim(strtoupper($config['configuration_key'])), $configValue);
```

Comment:
Probably should be trimmed up at line 27 before used in the multiple additional places in this function.